MonitorControl/Display.swift

```diff
@@ -3,7 +3,11 @@ import Cocoa
 import DDC
 import os.log
 
-class Display {
+class Display: Equatable {
```

Comment:
Extension should be separated from the main class body

Reply:
The Equatable part isn't actually used anymore, I left it in because it doesn't hurt to have it already, but I'll move it to an extension anyways.